plugins/mcreator-localization/lang/texts.properties

```diff
@@ -23,6 +23,8 @@ dialog.workspace_selector.delete_workspace.recent_list=Remove from recents list
 dialog.workspace_selector.delete_workspace.workspace=Delete workspace folder
 dialog.workspace_selector.delete_workspace.confirmation=<html><b>Are you sure you want to permanently delete {0}?</b>\
   <br><br>If you click <b>Yes</b>, all workspace files will be irreversibly deleted from the computer.
+dialog.workspace_selector.open_in_system_explorer = Open in system explorer
```

Comment:
`open_in_system_explorer = Open in system explorer`

Omit whitespace, check other lines in this file to see how this is formatted